tests/foreman/ui/test_hostunification.py

```diff
@@ -252,7 +435,40 @@ def test_positive_add_contents_to_unregistered_host(self):
         @assert: User get a warning:
         This Host is not currently registered with subscription-manager.
 
-        @caseautomation: notautomated
-
         @CaseLevel: System
         """
+        host = entities.Host()
+        host.create_missing()
+        os_name = u'{0} {1}'.format(
+            host.operatingsystem.name, host.operatingsystem.major)
+        with Session(self.browser) as session:
+            make_host(
+                session,
+                name=host.name,
+                org=host.organization.name,
+                parameters_list=[
+                    ['Host', 'Organization', host.organization.name],
+                    ['Host', 'Location', host.location.name],
+                    ['Host', 'Lifecycle Environment', ENVIRONMENT],
+                    ['Host', 'Content View', DEFAULT_CV],
+                    ['Host', 'Puppet Environment', host.environment.name],
+                    [
+                        'Operating System',
+                        'Architecture',
+                        host.architecture.name
+                    ],
+                    ['Operating System', 'Operating system', os_name],
+                    ['Operating System', 'Media', host.medium.name],
+                    ['Operating System', 'Partition table', host.ptable.name],
+                    ['Operating System', 'Root password', host.root_pass],
+                ],
+                interface_parameters=[
+                    ['Type', 'Interface'],
+                    ['MAC address', host.mac],
+                    ['Domain', host.domain.name],
+                    ['Primary', True],
+                ],
+            )
+            hostname = u'{0}.{1}'.format(host.name, host.domain.name)
+            self.contenthost.install_package(hostname, 'busybox')
+            self.assertIsNotNone(common_locators['alert.error'])
```

Comment:
`self.assertIsNotNone(common_locators['alert.error'])` locator will never be None :) maybe you wanted to wait for element with that locator?